src/System.Private.Xml/src/System/Xml/Xslt/XsltException.cs

```diff
@@ -9,21 +9,59 @@
 
 namespace System.Xml.Xsl
 {
+    [Serializable]
+    [System.Runtime.CompilerServices.TypeForwardedFrom("System.Xml, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
     public class XsltException : SystemException
     {
         private string _res;
+        private string[] _args;
         private string _sourceUri;
         private int _lineNumber;
         private int _linePosition;
 
+        // message != null for V1 & V2 exceptions deserialized in Whidbey
+        // message == null for created V2 exceptions; the exception message is stored in Exception._message
+        private string _message;
+
         protected XsltException(SerializationInfo info, StreamingContext context) : base(info, context)
```

Comment:
Odd that we removed the constructors that take args. I guess it is OK adding the dummy field.